java/pom.xml

```diff
@@ -231,6 +231,45 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>COPY_NVRTC</id>
```

Comment:
Curious why we need this change for this PR, or is this change also about static linking and marking dependencies as required and optional? My understanding of static linking is limited so pardon any naivety.

Reply:
Yes so we can statically link against cudart, but nvrtc which is the runtime compiler we cannot statically link against and it is tied to the cuda release version 10.02, 10.1, etc.  This copies it into the jar when we are doing static linking.